app/controllers/sign_up/completions_controller.rb

```diff
@@ -21,6 +21,10 @@ def update
       track_completion_event('agency-page')
       update_verified_attributes
       send_in_person_completion_survey
+      if user_session[:selected_email_id].nil?
```

Comment:
Nit / okay for follow-up: User session is global, and "selected email id" is rather ambiguous to what it refers to. I'd suggest qualifier this for what it's meant to be used for (e.g. `selected_email_id_for_linked_identity`).

Reply:
Ticket: [LG-14353](https://cm-jira.usa.gov/browse/LG-14353)